src/test/java/uk/gov/register/functional/AnalyticsFunctionalTest.java

```diff
@@ -38,56 +36,65 @@
         testEntry2 = TestEntry.anEntry(2, "{\"street\":\"" + testEntry2Key + "\",\"address\":\"12346\"}", "12346");
     }
 
-    @Rule
-    public RegisterRule register;
+    @ClassRule
+    public static RegisterRule register = new RegisterRule();
 
     @Before
     public void setup() {
         register.wipe();
-        register.mintLines(testEntry1.itemJson,
-                testEntry2.itemJson);
+        register.mintLines("address", testEntry1.itemJson, testEntry2.itemJson);
     }
 
     private final String targetUrl;
-    private final Boolean shouldIncludeAnalytics;
 
-    public AnalyticsFunctionalTest(String targetUrl, String trackingId, Boolean shouldIncludeAnalytics) {
-        this.register = createRegister(trackingId);
+    public AnalyticsFunctionalTest(String targetUrl) {
         this.targetUrl = targetUrl;
-        this.shouldIncludeAnalytics = shouldIncludeAnalytics;
     }
 
-    @Parameterized.Parameters(name = "{index} - url: {0} with code: {1} should include analytics: {2}")
+    @Parameterized.Parameters(name = "{index} - url: {0}")
     public static List data() {
-        List sourceData = new ArrayList();
+        List<Object[]> sourceData = new ArrayList<>();
```

Comment:
nope, it still needs to be a `List<Object[]>` but your syntax gives a `List<String>`